partner_firstname/views/base_config_view.xml

```diff
@@ -23,7 +23,7 @@
                                     icon="fa-play"
                                     type="object"
                                     help="Recalculate names for all partners. This process could take so much time if there are more than 10,000 active partners"
-                                    attrs="{'invisible': [('partner_names_order_changed', '=', False)]}"/>
+                                    attrs="{'invisible': [('partner_names_order_changed', '=', True)]}"/>
```

Comment:
@gdgellatly why invisible if changed? Shouldn't this be as it was before?

Reply:
@eLBati Because before it was broken and did the opposite of what was expected. It updated the partner names with the old value.  But you can't very well call set_values here as what if they changed something else, you can't write directly as that is counter to the rest of behaviour of settings options. so really the only way I could find to keep the function was to require an explicit write by the user and then an update.

Reply:
@eLBati in what way? it is in the commit message/PR already.

Reply:
@gdgellatly I meant a comment here in the XML file, to highlight the issue. Because I think it's not intuitive to allow to "Recalculate names for all partners" only when partners name order did not change

Reply:
@eLBati I ended up putting in an explicit write and reverting this view. 

This modules config.settings are not really well written, it really should have the config parameter as part of the module, with a noupdate, with a proper qualified name, and referenced by a config_parameter argument in the settings, Would get rid of a big percentage of the base settings code as well, but to keep backwards compatibility and same behaviour I just leave it as it is.  

I have no idea why it was decided to not set it in the first place and reimplement everything by hand. Maybe there is something non obvious I am missing.

Reply:
LGTM